examples/notebooks/beam-ml/run_inference_tensorflow.ipynb

```diff
@@ -332,7 +330,16 @@
     {
       "cell_type": "markdown",
       "source": [
-        "Instead of saving the entire model, you can just save the model weights for inference. This is slightly lightweight than saving and loading the entire model. However, you need to pass the function to build TensorFlow model to the `TFModelHandlerNumpy` / `TFModelHandlerTensor` class along with `ModelType.SAVED_WEIGHTS`."
+        "Instead of saving the entire model, you can just [save the model weights for inference](https://www.tensorflow.org/guide/keras/save_and_serialize#saving_loading_only_the_models_weights_values). This is useful in cases when you're using the model just for inference and won't need any compilation information or optimizer state. This also allows loading the weights with new model in case of transfer learning applications.\n",
```

Comment:
```suggestion
        "Instead of saving the entire model, you can [save the model weights for inference](https://www.tensorflow.org/guide/keras/save_and_serialize#saving_loading_only_the_models_weights_values). You can use this method when you need the model for inference but don't need any compilation information or optimizer state. In addition, when using transfer learning applications, you can use this method to load the weights with new models.\n",
```